runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -194,6 +195,42 @@ public boolean mergeSegments(Layout currentLayout)
         return true;
     }
 
+    /**
+     * Best effort attempt to removes a node from the layout.
+     *
+     * @param currentLayout the layout to remove the node from
+     * @param endpoint the node to remove
+     */
+    public void removeNode(@Nonnull Layout currentLayout,
+                           @Nonnull String endpoint) throws QuorumUnreachableException,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.QuorumUnreachableException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)